client/packages/invoices/src/Prescriptions/DetailView/Toolbar.tsx

```diff
@@ -69,11 +75,17 @@ export const Toolbar: FC = () => {
               }
             />
           </Box>
-          <Box display="flex" flex={1}>
+          <Box
+            display="flex"
+            flexDirection="column"
+            flex={1}
+            marginLeft={'8px'}
+          >
```

Comment:
```suggestion
          <Box display="flex" flexDirection="column" flex={1} marginLeft={3}>
```

two things - we need to remove hard coded pixel values for spacing and fonts in order to support responsive UI and I think the date field is too close to the patient field, given the gap between `Date:` and the input: it's easy to confuse the label & input association